use "standard" posix directory variables

r2evans opened the pull request:
- https://github.com/dspinellis/git-issue/issues/64#issuecomment-582281134
- https://www.gnu.org/software/make/manual/html_node/Directory-Variables.html
- https://www.gnu.org/software/automake/manual/html_node/Standard-Directory-Variables.html

dspinellis commented on git-issue.sh:
```diff
@@ -36,7 +36,7 @@ IFS=:
 # shellcheck disable=SC2086
 # Rationale: Word splitting not an issue
-LIB_PATH="$(dirname $0)/../lib:$LD_LIBRARY_PATH:/usr/local/lib:/usr/lib"
+LIB_PATH="$(dirname $0)/../libexec:$LD_LIBRARY_PATH:/usr/local/libexec:/usr/libexec"
```
Won't we hard-code `$(libexecdir)/git-issue` and loose the silliness of trying to guess the location?

r2evans replied:
oh yes, of course ...

r2evans commented:
If you remove all library detection, then won't that also remove the option for a personal installation?

    ### Personal installation
    Clone the repo and register a git alias to the `git-issue.sh` script:
    
    ```
    git config --global alias.issue '!'"${REPO_PATH}/git-issue.sh"
    ```

Perhaps it would be better to search `$(dirname $0)/../lib` only if the preferred path is not found?

dspinellis commented:
Right!  Keep only the `$0` check?

r2evans commented:
That is, perhaps we can trim the list of possibles down a bit, and make the test a little more specific:

```diff
 # Rationale: Word splitting not an issue
+LIB_PATH="$(dirname $0)/../libexec/git-issue:$LD_LIBRARY_PATH:/usr/local/libexec/git-issue:/usr/libexec/git-issue"
-LIB_PATH="/usr/local/libexec/git-issue:$(dirname $0)/../libexec/git-issue"
 if [ "x$GIT_ISSUE_LIB_PATH" != x ] ; then
   LIB_PATH="$GIT_ISSUE_LIB_PATH"
 fi
 for i in ${LIB_PATH} ; do
-  if [ -d "${i}" ] ; then
+  if [ -d "${i}" -a -f "${i}/import-export.sh" ] ; then
     MY_LIB="${i}"
```

We should probably

r2evans commented:
Sorry, I see the failed test, but I don't know what's causing it. It's likely in the hard-coded `LIB_PATH` checks, but since `.issues` is not being created with `test.sh` it all tumbles from there. I'll check back later.

r2evans commented:
I have a local fix that is creating `.issues` but is still marking the test as failed. The reason, I think, is that `gi.sh init` is creating a new log entry and leaving several files changed.

On my windows test machine, I see this artifact... any pointers?

```bash
$ git status
On branch fix/64_posix_makefile
nothing to commit, working tree clean

r2@d2sb2 MINGW64 ~/Projects/git-issue/_64_posix (fix/64_posix_makefile)
$ ./gi.sh init
Initialized empty issues repository in /c/Users/r2/Projects/git-issue/_64_posix/.issues

r2@d2sb2 MINGW64 ~/Projects/git-issue/_64_posix (fix/64_posix_makefile)
$ git status
On branch fix/64_posix_makefile
Changes not staged for commit:
  (use "git add/rm <file>..." to update what will be committed)
  (use "git restore <file>..." to discard changes in working directory)
        modified:   README.md
        deleted:    config
        deleted:    templates/comment
        deleted:    templates/description

no changes added to commit (use "git add" and/or "git commit -a")

r2@d2sb2 MINGW64 ~/Projects/git-issue/_64_posix (fix/64_posix_makefile)
$ git diff README.md
diff --git a/README.md b/README.md
index f440ebb..468599a 100644
--- a/README.md
+++ b/README.md
@@ -1,2 +1,539 @@
-This is an distributed issue tracking repository based on Git.
-Visit [git-issue](https://github.com/dspinellis/git-issue) for more information.
+[![Build Status](https://travis-ci.org/eellak/gsoc2019-git-issue.svg?branch=gsoc-2019)](https://travis-ci.org/eellak/gsoc2019-git-issue)
+# git-issue
+
+This is a minimalist decentralized issue management system based on Git,
+offering (optional) biderectional integration with GitHub and GitLab issue management.
+It has the following advantages over other systems.
.........
```

dspinellis commented:
Not sure what is going on with the failed test.  Regarding library detection, how about having the personal installation involve specifying the various directories through `=$(pwd)`?

r2evans commented:
(I apologize, I'm at a conference and unable to put any time into this for a bit...)
